content/en/docs/concepts/architecture/unknown-version-interoperability-proxy.md

```diff
@@ -0,0 +1,79 @@
+---
+reviewers:
+- jpbetz
+title: Unknown Version Interoperability Proxy
+content_type: concept
+weight: 220
+---
+
+<!-- overview -->
+{{< feature-state state="alpha"  for_k8s_version="v1.28" >}}
+
+Kubernetes {{< skew currentVersion >}} includes an alpha feature that lets one
+{{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}
+proxy a resource request to a different _peer_ API server. This is useful when there are multiple
+API servers running different versions of Kubernetes in one cluster (for example, during a long-lived
+rollout to a new release of Kubernetes).
```

Comment:
Remove "alpha here" since we already list the stability level above? (and avoid having one more thing to remember to update when we graduate this feature).

Reply:
This covers "what" and "when" Can we say more about "who", "how" and "why"? E.g. "this allows cluster administrators to configure highly available clusters to ... during upgrade by ... "?

Reply:
> Remove "alpha here" since we already list the stability level above? (and avoid having one more thing to remember to update when we graduate this feature).
Done

> Can we say more about "who", "how" and "why"?
Added

Reply:
> Remove "alpha here" since we already list the stability level above? (and avoid having one more thing to remember to update when we graduate this feature).

I suggested that text, and it's intentional. When we update to stable we'd omit the `{{< skew currentVersion >}}` shortcode. Someone looking at different docs versions will see “Kubernetes 1.28 ... alpha”, “Kubernetes 1.29 ... alpha”, “Kubernetes 1.30 ... beta” and finally ”Kubernetes lets you configure proxying&hellip;” once it's stable.

There's another reason. In code, it's good not to repeat ourselves. In tech docs, it can help to say things twice.

Reply:
sure, reverted back to original.